### PR TITLE
add openjdk-{11,17,20}-dbg subpackage

### PR DESCRIPTION
**Pull request by joshrwolf**
__relies on https://github.com/chainguard-dev/melange/pull/509__

adds an `openjdk-{11,17,20}-dbg` subpackages containing debug symbols

### CODE REVIEW

**Review by imjasonh — DISMISSED** [review]

**Review by imjasonh — DISMISSED** [review]

File: openjdk-11.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ environment:
       - lcms2-dev
       - openjdk-10-default-jvm
       - openjdk-10
+      - scanelf
```
Comment (joshrwolf) [inline openjdk-11.yaml:42]:
```suggestion
```

File: openjdk-17.yaml [hunk 2]
```diff
@@ -39,6 +39,7 @@ environment:
       - lcms2-dev
       - openjdk-16-default-jvm
       - openjdk-16
+      - scanelf
```
Comment (joshrwolf) [inline openjdk-17.yaml:42]:
```suggestion
```

File: openjdk-20.yaml [hunk 3]
```diff
@@ -39,6 +39,7 @@ environment:
       - lcms2-dev
       - openjdk-19-default-jvm
       - openjdk-19
+      - scanelf
```
Comment (joshrwolf) [inline openjdk-20.yaml:42]:
```suggestion
```

**Comment by joshrwolf** [comment]
check update is expected to fail due to mangled java versions

**Review by imjasonh — APPROVED** [review]

**Pull request merged**